<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>üí∞ Money Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxInput">
      <layout class="QGridLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDate">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCategory">
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxCategory">
         <item><property name="text"><string>Food</string></property></item>
         <item><property name="text"><string>Transport</string></property></item>
         <item><property name="text"><string>Salary</string></property></item>
         <item><property name="text"><string>Others</string></property></item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAmount">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditAmount">
         <property name="placeholderText">
          <string>Rp. 1.000</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxType">
         <item><property name="text"><string>Income</string></property></item>
         <item><property name="text"><string>Expense</string></property></item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelNotes">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="textEditNotes"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>‚ûï Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="columnCount">
       <number>5</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Date</string>
        <string>Category</string>
        <string>Amount</string>
        <string>Type</string>
        <string>Notes</string>
       </stringlist>
      </property>
     </widget>
    </item>
   </layout>
  </widget>

  <!-- Menu Bar -->
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>üìÅ File</string>
    </property>
    <addaction name="actionExport_CSV"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>‚ùì Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="actionClose"/> <!-- Exit langsung di menubar -->
  </widget>

  <widget class="QStatusBar" name="statusbar"/>

  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>‚ùå Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
